prometheus/src/main/scala/filodb/prometheus/query/PrometheusModel.scala

```diff
@@ -91,8 +91,12 @@ object PrometheusModel {
   /**
     * Used to send out HTTP response
     */
-  def toPromResult(srv: SerializableRangeVector): Result = {
-    Result(srv.key.labelValues.map { case (k, v) => (k.toString, v.toString)},
+  def toPromResult(srv: SerializableRangeVector, verbose: Boolean): Result = {
+    val tags = srv.key.labelValues.map { case (k, v) => (k.toString, v.toString)} ++
+                (if (verbose) Map("shards" -> srv.key.sourceShards.toString(), "partIds" -> srv.key.partIds.toString())
```

Comment:
Should we follow the reserved keyword label syntax.

`_shards_`
`_partids_`

This way, it will standout clearly and will not conflict with any other user labels.

Reply:
Agreed. Fixed.